### ISSUE TITLE
create full documentation of crossreferenced members to their QR codes

### ISSUE THREAD

**Issue by chiselinc** [DecentralVan/decentral#86]
if someone does this i will keep it updated moving forward, as discussed-- including maintaining two pronged [internal editable and public facing static] drafts

SRSLY someone please do this, it's important and i've already committed to like ten tasks

@paudley you're super organized, maybe you can obtain the existing info through @astupidmoose @cammyjee and @TRhode and put one together?

**Comment by findkiko** [comment]
There's  a bunch of issues conflated here, not quite sure what you want/what the specific problem is.

I've added spare receiving addresses to the end of the member list to reduce the new-address bottleneck.

re: security I have old snapshots of the list that would help against malicious mutation, i encourage others to do the same.

All the assignees here have access to lookup addresses, not sure what else you want re: cross-referencing.

**Comment by Latrasis** [comment]
Slightly confused as well, can you give more context?

**Comment by chiselinc** [comment]
this was based on something @paullucas was talking about at the meeting, i believe-- basically making it easier to cross reference active members with their addresses etc.

sounds like you've got the infrastructure set, and thanks for generating some new ones. i think what i would find most helpful, and this regards the accounting spreadsheet, is for us to come to some sort of consensus of how to organize members that are active, members that we know for a fact are gone/on hiatus, and members who we need to follow up with to see what their status is. cos as it stands the spreadsheet, at least, has everyone all mixed together and i have to look at who hasn't payed and then look over and be like "oh duh cos that person's gone"

basically i suck at record keeping of this nature so i'd prefer not to take on the organizing aspect, but if someone can give a bit more structure-- solidly active members, "hiatus", "archived members", and members who have been inconsistent in payment, it will be easier to create a structure around sending out reminder emails etc.

additionally, separating out the "gone" people will help us keep track of our ability to pay the rent-- i had thought we were scaling great re:new members then kiko pointed out that we've simultaneously been bleeding old members.

so yeah, brain dump, but the original QR code aspect of this IIRC was a comment from paul at the meeting so perhaps he can clarify

**Comment by chiselinc** [comment]
okay since this is clearly a disorganized flow of thought, and i didn't summarize the meeting discussion as well as i assumed, i'll close this for now and we'll revisit accounting/operational organization with another approach at the next meeting
